n3fit/src/n3fit/model_gen.py

```diff
@@ -26,7 +26,7 @@
     base_layer_selector,
 )
 from n3fit.backends import operations as op
-from n3fit.backends import regularizer_selector
+from n3fit.backends import regularizer_selector as reg_sec
```

Comment:
Why this? Presumably you called it regularizer_selector because it's more descriptive, which is now lost